python/code_to_gast/py_loop.py

```diff
@@ -41,9 +41,14 @@ def for_range_statement_to_gast(node):
         gast["type"] = "forRangeStatement"
         ast.dump(node)
         args = node.iter.args
-        step_num = get_step_num(args)
-        start_num = arg_node_to_num(args[0])
-        end_num = arg_node_to_num(args[1])
+        if len(args) == 1:
```

Comment:
Can we add a comment here or wherever this logic ends up living showing example code snippets that lead to each branch?